clients/mobile/src/setup/theme/context/ThemeContext.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { createContext, useState, useEffect, ReactNode } from 'react'
+import { useColorScheme } from 'react-native'
+import { lightTheme, darkTheme, Theme } from '..'
+
+export const ThemeContext = createContext<Theme | null>(null)
+
+interface ThemeProviderProps {
+  children: ReactNode;
+}
+
+export const ThemeProvider: React.FC<ThemeProviderProps> = ({ children }) => {
+  const systemTheme = useColorScheme()
+  const [theme, setTheme] = useState<Theme>(systemTheme === 'dark' ? darkTheme : lightTheme)
```

Comment:
this should be const, the theme name

Reply:
Don't understand this vai should I change this to const or theme to `themeName`. But as I can see this is already const.

Reply:
The keyword dark should be a constant